config.py

```diff
@@ -45,5 +45,6 @@
     'name': '{} Keras Model'.format(MODEL_NAME),
     'description': '{} Keras model trained on ImageNet'.format(MODEL_NAME),
     'type': 'image_classification',
-    'license': '{}'.format(MODEL_LICENSE)
+    'license': '{}'.format(MODEL_LICENSE),
```

Comment:
(Not strictly relevant to this PR) Do we need a `format` here? Can we simply do
```suggestion
    'license': MODEL_LICENSE,
```